medical_patient_disease/models/medical_patient_disease.py

```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class MedicalPatientDisease(models.Model):
+    _name = 'medical.patient.disease'
+    _description = 'Medical Patient Disease'
+
+    treatment_description = fields.Char()
+    short_comment = fields.Char()
+    name = fields.Char(
+        compute='_compute_name',
+        store=True,
+    )
+    expire_date = fields.Datetime(
```

Comment:
Wouldn't it be nicer to add `string='Expiration Date'` so it looks better on the front end? The field is visible in a view.